packages/formdescriptioneditors/frontend/sirius-components-formdescriptioneditors/src/ButtonWidget.tsx

```diff
@@ -45,11 +75,14 @@ export const ButtonWidget = ({ widget, selection }: WidgetProps) => {
       </Typography>
       <Button
         data-testid={widget.label}
+        classes={{ root: classes.style }}
         variant="contained"
-        color="primary"
         onFocus={() => setSelected(true)}
         onBlur={() => setSelected(false)}
         ref={ref}>
+        {widget.imageURL?.length > 0 ? (
+          <img className={classes.icon} width="16" height="16" alt={widget.label} src={httpOrigin + widget.imageURL} />
```

Comment:
I can't get the image preview to work. My widget correctly displays my image in an actual instance, but not on the preview.
I tried with a fixed, absolute HTTP URL, and a relative URL to a custom image I uploaded.

Reply:
Done